src/Symfony/Bundle/FrameworkBundle/CacheWarmer/RouterCacheWarmer.php

```diff
@@ -26,12 +26,9 @@
  */
 class RouterCacheWarmer implements CacheWarmerInterface, ServiceSubscriberInterface
 {
-    private ContainerInterface $container;
-
-    public function __construct(ContainerInterface $container)
+    // As this cache warmer is optional, dependencies should be lazy-loaded, that's why a container should be injected.
```

Comment:
Should this inside a docblock on the constructor (same for the other cache warmers)?

Reply:
You did on TemplateCacheWarmer indeed, let's be consistent